docs/sources/release-notes/v2.2.md

```diff
@@ -0,0 +1,43 @@
+---
+title: "Grafana Mimir version 2.2 release notes"
+menuTitle: "V2.2 release notes"
+description: "Release notes for Grafana Mimir version 2.2"
+weight: 300
+---
+
+# Grafana Mimir version 2.2 release notes
+
+Grafana Labs is excited to announce version 2.2 of Grafana Mimir, the most scalable, most performant open source time series database in the world.
+
+Highlights include the top features, enhancements, and bugfixes in this release. If you are upgrading from Grafana Mimir 2.1, there is migration-related information as well.
+For the complete list of changes, see the [Changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Support for ingesting out-of-order samples**: Grafana Mimir includes new, experimental support for ingesting out-of-order samples.
+  This support is configurable, with users able to set how far out-of-order Mimir will accept samples on a per-tenant basis.
+  Note that this feature still needs a heavy testing, and is not production-ready yet.
+  **TODO: Link to documentation after https://github.com/grafana/mimir/pull/2183 is merged.**
+
+- **Error messages**: The error messages that Mimir reports are more human readable, and the messages include error codes that are easily searchable.
```

Comment:
Let's not call it "codified errors" but "errors catalog". @zenador can you open a PR to rename it in `mimir-runbooks/_index.md`, please? Then here in the release note we could say we introduced a Mimir errors catalog.